modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -190,7 +190,7 @@ const DatePicker = ({
 					}
 
 					if (moment(value).isValid()) {
-						onChange(moment(value).format(dateMask));
+						onChange(moment(value).format(dateMask.toUpperCase()));
```

Comment:
Good idea with this.

I think ideally we might swap out moment for date-fns here in the future

Reply:
Yeah, I'm seeing how we can try to track this, we may have to create an epic to remove the use of moment from DXP.